delivery_recipe/help-cards/help-card-user-stories.md

```diff
@@ -0,0 +1,60 @@
+# User Stories
+
+When we are capturing the requirements for a feature we want to do it in a way that we can all \(developers and non-developers, client-side and supplier-side\) have a useful conversation about the need that the requirement addresses, and possible ways to deliver it.
+
+We use a standard way to set out these needs, and we call the format a 'user story'.
+
+Two words are key in this phrase — user, and story:
+
+1. User: who actually needs this thing, our main character in the story
+2. Story: telling a story about what they do and why
+
+In this story we don't need to capture every single detail about the need this user has. The intention is for user stories to be a 'placeholder for a conversation'. We're capturing the need in its bare details, and we can then discuss it later when we come to plan its implementation in a sprint.
+
+## Format
+
+We write user stories in a standard format so that we can all quickly approach any of these written up needs, understand how they are expressed, and take part in a conversation about them.
+
+At it's most basic this format is:
+
+> As a &lt;who&gt; I want &lt;what&gt; so that &lt;why&gt;
+
+Which can also be thought of as:
+
+> As a &lt;user role&gt; I want &lt;need&gt; so that &lt;benefit&gt;
+
+## Start With User Needs
+
+User stories are not simply a new way for stakeholders to express the same stuff that would previously have gone in a big requirements spreadsheet. They are the output of a different way of approaching defining the needs of a project.
+
+First comes the business goals. From that we identify what types of people the organisation needs to interact with to meet the goals. These are the user groups, and we need to understand their needs, in support of the business goals, and deliver them.
+
+So we conduct user research, and user testing if there is an existing service, and it is from this that we gain the insights that we write as user stories.
```

Comment:
Once it is published, can I open an issue and push to you to fill in this gap? At the moment this is just moving content that was hosted across different pages to be on one handy helpcard. It might even merit its own helpcard.